docs/init.md

```diff
@@ -0,0 +1,14 @@
+---
+id: init
+title: bolt init
+---
+
+---
```

Comment:
What are these for?

Reply:
These are to have line after heading, 
![image](https://user-images.githubusercontent.com/9667784/35217409-4ae2dc66-ffbf-11e7-9e25-65d549f5367b.png)